lib/adapter/codecept.js

```diff
@@ -76,11 +77,22 @@ function CodeceptReporter(config) {
     if (!global.testomatioDataStore) global.testomatioDataStore = {};
     // reset steps
     global.testomatioDataStore.steps = [];
+
+    // TODO: use suite + title
+    const testId = parseTest(test.tags);
+    const logger = new Logger({ adapter: 'codeceptjs', testId });
```

Comment:
do not recreate logger, logger is a singleton

```suggestion
    const loggingContext = parseTest(tags.test) || `test/${test.fullTitle()}`
    const logger = logger.getInstance();
    logger.startContext(loggingContext)
```